chat-widget/src/components/prechatsurveypanestateful/PreChatSurveyPaneStateful.tsx

```diff
@@ -117,7 +117,12 @@ export const PreChatSurveyPaneStateful = (props: IPreChatSurveyPaneStatefulParam
                 const current = children[index];
                 if (current && current.className == HtmlAttributeNames.adaptiveCardTextBlockClassName) {
                     value = current.innerHTML;
```

Comment:
Is this line still needed then? Is there a case where current doesn't have a <p> child?

Reply:
Added `childElementCount > 0`